frontend/public/locales/en/translation.json

```diff
@@ -2128,6 +2128,7 @@
     "SSH private key": "SSH private key",
     "SSH public key": "SSH public key",
     "Standalone": "Standalone",
+    "Standalone clusters will display the namespace used by the ManagedCluster resource. Hosted clusters will display the hosting namespace when the status is \"Pending import\" and the ManagedCluster namespace when the status is \"Ready\".": "Standalone clusters will display the namespace used by the ManagedCluster resource. Hosted clusters will display the hosting namespace when the status is \"Pending import\" and the ManagedCluster namespace when the status is \"Ready\".",
```

Comment:
@KevinFCormier what do you think about this? Does this make sense?

Reply:
Are you asking about the wording or the behaviour? Wording-wise I think it's good, although I thought @chenz4027 made a change so that hosted clusters will show "Importing" status instead of showing "Pending import"?

Behaviour-wise it's a little confusing, but I'm not sure that we can do any better. @rawagner put together this document outlining the issues we have with the various namespace-scoped and cluster-scoped resources that are used for the cluster list: https://docs.google.com/document/d/1tXTTZC4Waqk8glc3GZlNuLcMl9aE90KfqO_8D7PAzaQ/edit?pli=1#heading=h.vmfpofhkb0mi 

@rawagner, after your change to include the namespace in the route, how is this namespace used to associate the resources?

Reply:
Thanks @KevinFCormier ! Yep, I'm just asking about the wording. The tooltip is suppose to help the user understand why we have the current behaviour before and after import. In the latest console code, I still see this so I used "Pending import":

![image](https://user-images.githubusercontent.com/38960034/206301000-f741576b-4ff4-4210-9a79-77e1db44f038.png)

Reply:
@fxiang1 I see this is still waiting to merge. @chenz4027 is "Pending import" correct?

Reply:
@KevinFCormier Thanks Kevin for pinging me. Yep so before the user clicks "import", it will show pending import. Once the user click the import button, it will switch to "importing status".

Reply:
Thanks for clarifying.